test/e2e/storage/testsuites/topology.go

```diff
@@ -161,6 +163,13 @@ func (t *topologyTestSuite) defineTests(driver TestDriver, pattern testpatterns.
 			cleanup(l)
 		}()
 
+		// If possible, exclude one topology, otherwise allow them all
+		excludedIndex := -1
+		if len(l.topologies) > dInfo.NumAllowedTopologies {
+			excludedIndex = rand.Intn(len(l.topologies))
+		}
+		allowedTopologies := t.setAllowedTopologies(l.resource.sc, l.topologies, excludedIndex)
```

Comment:
nit: more specific names than `allowedTopologies` and `l.topologies` to distinguish between the two

Reply:
renamed to `l.allTopologies`